tests/test_pipeline.py

```diff
@@ -4,11 +4,30 @@
 
 import tempfile
 
+import spacy
+
 from textpipe.pipeline import Pipeline
 
 TEXT = 'Test sentence for testing'
-PIPELINE = [('Raw',), ('NWords',), ('Complexity',), ('CleanText',)]
-PIPE = Pipeline(PIPELINE)
+ents_model_nl = spacy.blank('nl')
+ents_model_en = spacy.blank('en')
+model_path_nl = tempfile.mkdtemp()
+model_path_en = tempfile.mkdtemp()
+ents_model_nl.to_disk(model_path_nl)
+ents_model_en.to_disk(model_path_en)
+PIPELINE = [('Raw',), ('NWords',), ('Complexity',), ('CleanText',),
+            ('Entities', {'model_mapping': {'nl': 'ents', 'en': 'other_identifier'}})]
+PIPE = Pipeline(PIPELINE,
+                models=[('ents', 'nl', model_path_nl), ('other_identifier', 'en', model_path_en)],
+                allowed_languages=('en', 'nl'))
+
+
+def test_load_custom_model():
+    """
+    The returned dictionary should have the same length as the pipeline
+    """
```

Comment:
This doesn't seem to test this: `The returned dictionary should have the same length as the pipeline`